exercise_perseus_render/assets/src/vue/index.vue

```diff
@@ -29,7 +29,7 @@
         :log="recentAttempts"
       >
       </exercise-attempts>
-      <p class="message">Get <b>{{passRatioM}}</b> check marks showing up!</p>
+      <p class="message">GOAL: Get <b>{{passRatioM}}</b> check marks showing up!</p>
```

Comment:
At some point we should update this to use a pluralized translation string.

See an example here: https://github.com/learningequality/kolibri/blob/10aac7d3343cd0005b9cbdc5dab76513bcfeb9c5/kolibri/plugins/coach_tools/assets/src/vue/reports/summary-section/index.vue

Also @christianmemije has some experience with the syntax.